src/encoded/static/components/medicationChart.js

```diff
@@ -22,16 +25,38 @@ class MedicationChart extends React.Component {
     this.drugNames = [];
     this.series = [];
     this.scaleYIndex = 0;
+    this.zoomIn = this.zoomIn.bind(this);
+    this.zoomOut = this.zoomOut.bind(this);
+
   }
   render() {
     return (
       <div className="flex-container" >
+        <div className="chart-menu" >
+          <div className="chartboxes pt-2" >
+            <div className="mb-2">
+              <button type="button" className="btn btn btn-danger" onClick={this.zoomIn} title="Zoom in" aria-label="Zoom in"><FontAwesomeIcon icon={faSearchPlus} size="2x" /></button>

```

Comment:
Different className for zoom in and zoom out buttons compare to zoom in and zoom out button in patientChart.js. We need to discuss what className to use to keep consistency.

Reply:
Add to Radiation and standardize across all objects